content/github/teaching-and-learning-with-github-education/applying-for-a-student-developer-pack.md

```diff
@@ -1,5 +1,5 @@
 ---
-title: Applying for a student developer pack
+title: Applying for the Student Developer Pack
```

Comment:
```suggestion
title: Applying for a Student Developer Pack
```
Oh, just noticed! Can we leave this as `a`? If we change the wording in the title we'll need to make some additional changes to the index.md, add redirects, and update anywhere we link to this article.

We can do that, but it'll be a bigger project and would be better suited to opening an issue in the internal repo for a writer to take care of ✨